src/app.js

```diff
@@ -1,21 +1,20 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const parts = normalizedURL.pathname.slice(1).split('/');
+  const query = Object.fromEntries(normalizedURL.searchParams.entries());
```

Comment:
If we use Object.fromEntries we will get just the last parameter in search if it will be repeated, maybe it is possible to find the way to fix it?)

Reply:
Agree, this can be a problem, we don't have such test for this case in this task, but it's better to find another solution